cv/qc-cv.obo

```diff
@@ -539,14 +564,1316 @@ is_a: QC:4000009 ! ID based
 is_a: QC:4000022 ! chromatogram metric
 relationship: has_relation QC:4000014 ! QC metric relation: multiple runs
 
-[Typedef]
-id: has_relation
-name: has_relation
+[Term]
+id: QC:4000107
+name: Column type
+def: "The parent to the group of column definitions for name, content, and units for table type." [PSI:QC]
+relationship: has_relation QC:4000006 ! table
+
+[Term]
+id: QC:4000108
+name: Retention time
+def: "The column contains retention time points in seconds." [PSI:QC]
+is_a: QC:4000107 ! Column type
+property_value: has_units UO:0000010
+
+[Term]
+id: QC:4000109
+name: Relative intensity
+def: "The column contains relative intensity values of one type." [PSI:QC]
+is_a: QC:4000107 ! Column type
+property_value: has_units MS:1000043
+
+[Term]
+id: QC:4000110
+name: Sequence coverage
+def: "The column contains sequence coverage values in percent, e.g. from proteins." [PSI:QC]
```

Comment:
make it explicit that allowed values are in range [0,100]?
Otherwise people might use [0,1]